tensornetwork/network_operations.py

```diff
@@ -29,13 +29,16 @@
 
 
 def norm(node: BaseNode) -> Tensor:
-  """The L2 norm of `node`
+  """The L2 norm of `Node`
+
   Args:
     node: A `BaseNode`. 
+
   Returns:
-    The L2 norm 
+    The L2 norm.
+
   Raises:
-    AttributeError: If `node` has no `backend` attribute.
+    AttributeError: If `Node` has no `backend` attribute.
```

Comment:
since `node` is an argument here, don't capitalize it.

Reply:
This appears several places. Be sure to update all of them!

Reply:
Done